pkg/apis/fleet.cattle.io/v1alpha1/values.go

```diff
@@ -1,27 +0,0 @@
-package v1alpha1
-
-import (
-	"encoding/json"
-
-	"github.com/rancher/wrangler/v2/pkg/data/convert"
-)
-
-type GenericMap struct {
```

Comment:
I renamed the file to match the type name, but only `DeepCopyInto` was modified.